docs/source/contributing.rst

```diff
@@ -116,6 +123,9 @@ To list Optimism contracts' addresses, run::
     sh ./docker/optimism/optimism.sh addresses
 
 
+Note: Networks are connected as submodules. Consider this when you clone/pull.
+
+
```

Comment:
Could you please explain what this is about?

Reply:
I believe he is referring to the extra step required to pull the submodules after doing a `git clone`: 
`git submodule update --init` OR `git submodule --init`.

Anyways, i believe we should add the `git submodule update --init` as a required command before starting any local rollup instance.